### ISSUE TITLE
DateTime.AddDays() results are not comparable with DateTime instances

### ISSUE THREAD

**Issue by fabriciomurta** [bridgedotnet/Bridge#3777]
The result of a DateTime instance's `AddDays()` call triggers an exception if compared to the initial DateTime instance value. This is probably worth double-checking when the fix is applied, if it no longer happens also for other operations like `AddMinutes()`, and so on.

### Steps To Reproduce

https://dev.deck.net/017a4e267727bec86a8bb2b40867833e

```csharp
public class Program
{
    public static void Main()
    {
        DateTime d = DateTime.Now;
        DateTime d2 = d.AddDays(3);
        
        Console.WriteLine(DateTime.Compare(d, d2));
    }
}
```

### Expected Result

According to this dotnet fiddle: https://dotnetfiddle.net/kh2znn

```js
-1
```

### Actual Result

```
System.Exception: TypeError: Cannot read property 'low' of null
    at o (https://dev.deck.net/resources/js/bridge/bridge.min.js?17.4.0-201810040712:7:127838)
    at i.n.compare (https://dev.deck.net/resources/js/bridge/bridge.min.js?17.4.0-201810040712:7:130212)
    at System.Int64.compareTo (https://dev.deck.net/resources/js/bridge/bridge.min.js?17.4.0-201810040712:7:136434)
    at Object.compare (https://dev.deck.net/resources/js/bridge/bridge.min.js?17.4.0-201810040712:7:19499)
    at Object.compare (https://dev.deck.net/resources/js/bridge/bridge.min.js?17.4.0-201810040712:7:19046)
    at Function.Main (https://dev.deck.net/RunHandler.ashx?h=8703483:12:45)
    at https://dev.deck.net/resources/js/bridge/bridge.min.js?17.4.0-201810040712:7:42674
    at HTMLDocument.i (https://dev.deck.net/resources/js/bridge/bridge.min.js?17.4.0-201810040712:7:6515)
    at i (https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js:2:27151)
    at Object.add [as done] (https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js:2:27450)
```

### See also
[DateTime.Compare exception](https://forums.bridge.net/forum/bridge-net-pro/bugs/5979) Bridge forums' thread.

**Comment by geoffreymcgill** [comment]
This scenario has been fixed. After another review and final testing, the fix will be included in the upcoming Bridge 17.8.0 release.

You can confirm with the following dev.deck.net sample:

https://dev.deck.net/017a4e267727bec86a8bb2b40867833e

**Comment by fabriciomurta** [comment]
A full test over all time zones have been undegone and all looks good, so the fix for this will be available next Bridge release!
